### PR TITLE
[Add] Additional Python Wrapper (Release 2.7.0)

### PR DESCRIPTION
**Pull request by oliveralka**
# Description

As discussed, we would like to add additional wrappers with examples for Release 2.7.0

BasicProteinInferenceAlgorithm.pxd
BayesianProteinInferenceAlgorithm.pxd
MzQCFile.pxd
FeatureFinderIdentificationAlgorithm.pxd

# Checklist:
- [X] Make sure that you are listed in the AUTHORS file
- [X] Add relevant changes and new features to the CHANGELOG file
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] New and existing unit tests pass locally with my changes
- [X] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

# How can I get additional information on failed tests during CI:
If your PR is failing you can check out 
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. If you click in the column that lists the failed tests you will get detailed error messages.

# Note:
- Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

**Comment by oliveralka** [comment]
FeatureFinderIdentificationAlgorithm.pxd
`ProgressLogger& getProgressLogger() { return prog_log_; }`
`ProgressLogger getProgressLogger() nogil except + # wrap-ignore`
Is not wrapped, since it lead to a funky error:

```
Error compiling Cython file:
------------------------------------------------------------
...
    
    def getProgressLogger(self):
        """Cython signature: ProgressLogger getProgressLogger()
        Return progress logger
"""
        cdef _ProgressLogger * _r = new _ProgressLogger(self.inst.get().getProgressLogger())
                                                      ^
------------------------------------------------------------
```

**Comment by jpfeuffer** [comment]
I don't think you can return references. You would need to return either a pointer or a copy. But I don't think anyone needs the ProgressLogger (since it's only for changing it). One can also just create a new one and overwrite the old. I don't even know if progress logging works reliably in python.

File: src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.h [hunk 1]
```diff
@@ -82,6 +82,8 @@ class OPENMS_DLLAPI FeatureFinderIdentificationAlgorithm :
   PeakMap& getMSData() { return ms_data_; }
```
Comment (oliveralka) [inline src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.h:82]: Should the definition of the getters and setter be moved to the source file?

Comment (timosachsenberg) [reply src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.h:82]: yes please

**Review by poshul — APPROVED** [review]
Looks like a good positive addition of more wrappers.

**Review by timosachsenberg — APPROVED** [review]
Very nice addition and thanks for the fixes.

**Pull request merged**